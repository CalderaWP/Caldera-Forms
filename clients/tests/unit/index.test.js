describe( 'Testing', ()=> {
    it( 'Works', ()=> {
        const roy= true;
        expect(roy).toBe(roy);
    })
    it( 'Can snapshot', ()=> {
        const mike = {
            roy:true
        };
        expect(JSON.stringify(mike)).toMatchSnapshot();
    })
})