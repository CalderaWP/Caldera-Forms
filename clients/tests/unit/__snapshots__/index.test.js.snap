// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Can snapshot 1`] = `"{\\"roy\\":true}"`;
